main/src/ui/main_window.vala

```diff
@@ -55,6 +55,17 @@ public class MainWindow : Gtk.Window {
     }
 
     private void setup_unified() {
+#if _WIN32 // WIN32 seems to not support exporting these symbols
```

Comment:
This shouldn't be needed and is a sign that the build is not correct. I am very certain that exporting symbols is possible, if it's not, you might be using the wrong linker.

Reply:
Done, I forgot to prefix the linker option with `-Wl`... that is why it wasn't working 🙄